src/test/java/com/dsrm/dsrmbackend/admin/AdminRoomTypeControllerTests.java

```diff
@@ -72,4 +73,27 @@ public void tryToAddNamelessRoomType() throws Exception {
                 .andExpect(status().isBadRequest())
                 .andExpect(jsonPath("$").value("name must not be blank"));
     }
+
+    @Transactional
+    @Test
+    public void deleteExistentRoomType() throws Exception {
+        this.mockMvc.perform(delete("/admin/room-types/2")
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isNoContent())
+                .andExpect(jsonPath("$.name").value("Sala telewizyjna"));
+        this.mockMvc.perform(get("/room-types/2")
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isNotFound());
+        this.mockMvc.perform(get("/rooms/3")
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.roomType").value(nullValue()));
+    }
+
+    @Test
+    public void tryToDeleteNonExistentRoomType() throws Exception {
+        this.mockMvc.perform(delete("/admin/room-types/2222")
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isNotFound());
+    }
 }
```

Comment:
as above + I think the test is redundant since we are checking a non-existent record anyway